src/screens/EventDetailsScreen/component.js

```diff
@@ -19,10 +20,10 @@ import LayoutColumn from "../../components/LayoutColumn";
 import ShadowedScrollView from "../../components/ShadowedScrollView";
 import SectionDivider from "../../components/SectionDivider";
 import PerformanceList from "../../components/PerformanceList";
-import { groupPerformancesByPeriod } from "../../selectors/events";
+// Move this to performance selector
```

Comment:
It is already in the performance selectors, isn't it?

Reply:
Yep, originally it wasn't. This was a note for me to do that work that I forgot to remove